### PR TITLE
Backport schema generator changes and polishing 💅

### LINKED ISSUE #6: Update schema generator
Backport changes from https://github.com/confio/cosmwasm-examples/pull/12, at least `schema.rs`.

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #6

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
-name = "{{project-name}}"
+name = "insert_project_name_here"
```
Comment (webmaster128) [inline Cargo.toml]: According to https://github.com/confio/cosmwasm-template/issues/7#issuecomment-567897838, this change must be reverted, right?

I did it in order to be able to test my changes.

File: .circleci/config.yml [hunk 2]
```diff
@@ -23,8 +23,8 @@ jobs:
           env: RUST_BACKTRACE=1
           command: cargo test --locked
       - run:
-          name: Ensure schema generation works
-          command: cargo schema
+          name: Build and run schema generator
+          command: cargo schema --locked
```
Comment (ethanfrey) [inline .circleci/config.yml]: This is good. I would be interested in adding a command to ensure this doesn't changes anything either (to be tested on escrow/erc20 first). Something like following this with `git diff schema` (but that only outputs the diff, if any, doens't have a useful return code)

Comment (webmaster128) [reply .circleci/config.yml]: okay, will do that. I have a command somewhere ready for copy/paste.

Comment (webmaster128) [reply .circleci/config.yml]: Done in https://github.com/confio/cosmwasm-examples/pull/13 and https://github.com/confio/cosmwasm-template/pull/8/commits/26102761138ed83e23e9bb7a669a0a805a426171

**Pull request merged**